python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -1234,6 +1218,8 @@ def do_evaluate(
 
     def collect_agg(self, *, depth: int) -> AggInfo:
         """Collect information about aggregations in groupbys."""
+        if self.name in {"unique", "drop_nulls"} | self._supported_cum_aggs:
+            raise NotImplementedError(f"{self.name} in groupby")
```

Comment:
Are the `unique` and `drop_nulls` cases already tested? What about the `cum_aggs`? The coverage check may be passing despite one or the other of these tests being untested since the line would still be hit.

Reply:
Test is introduced here:

https://github.com/rapidsai/cudf/pull/16719/files#diff-c0c5f4316fd6be501dbed4d33b65af567f2189868ffc54257f7493c645ae20b3R183

Reply:
Ah I didn't see that one, thanks.